src/Logic.hs

```diff
@@ -272,16 +272,19 @@ handleCommentAdded
   -> Text
   -> ProjectState
   -> Action ProjectState
-handleCommentAdded triggerConfig projectConfig pr author body =
+handleCommentAdded triggerConfig projectConfig pr author body state =
   let
     -- If the message was a valid merge command, then it was approved by the
     -- author of the message. Otherwise do nothing.
-    update pullRequest =
-      if (isMergeCommand triggerConfig body) && (isReviewer projectConfig author)
-        then pullRequest { Pr.approvedBy = Just author }
-        else pullRequest
+    update pullRequest = pullRequest { Pr.approvedBy = Just author }
+    isValidApproveCommand = (isMergeCommand triggerConfig body) && (isReviewer projectConfig author)
   in
-    pure . Pr.updatePullRequest pr update
+    if isValidApproveCommand
+      then do
+        leaveComment pr "Added PR to merge queue"
```

Comment:
I was thinking we could add who approved it an what its position in the queue to the comment too. We can inspect the queue with `filterPullRequestsBy` to filter on approved-but-not-conflicted PRs, and then filter the remaining ids on `< pr`, to count how many there are before this one.